charts/spire/charts/spire-server/values.yaml

```diff
@@ -327,6 +327,37 @@ upstreamAuthority:
       address: ""
       ## @param upstreamAuthority.spire.server.port Port for upstream Spire server
       port: 8081
+  vault:
+    ## @param upstreamAuthority.vault.enabled Enable Hashicorp Vault as upstream CA
+    enabled: false
+    ## @param upstreamAuthority.vault.vaultAddr The URL of the Vault server. (e.g., https://vault.example.com:8443/)
+    vaultAddr: ""
+    ## @param upstreamAuthority.vault.namespace Name of the Vault namespace. This is only available in the Vault Enterprise.
+    namespace: ""
+    ## @param upstreamAuthority.vault.pkiMountPoint Name of the mount point where PKI secret engine is mounted
+    pkiMountPoint: "pki"
+      ## @param upstreamAuthority.vault.insecureSkipVerify If true, caCert options are ignored and Spire accepts any server certificates claiming to be Vault
+    insecureSkipVerify: false
+    caCert:
+      ## @param upstreamAuthority.vault.caCert.type Type of resource representing the Vault server certificate, options are 'Secret' or 'Configmap', the item must be named `ca.crt`
+      type: ConfigMap
```

Comment:
shall we default this to Secret instead, that would make it work out of the box when pointing to a cert-manager secret.